benchmark/cublas/CMakeLists.txt

```diff
@@ -0,0 +1,51 @@
+if(BLAS_VERIFY_BENCHMARK)
+  find_package(SystemBLAS REQUIRED)
+endif()
+
+enable_language(CUDA) 
+find_package(CUDAToolkit)
+
+set(sources
+  # Level 1 blas
+  blas1/asum.cpp
+  blas1/axpy.cpp
+  blas1/dot.cpp
+  blas1/iamax.cpp
+  blas1/iamin.cpp
+  blas1/nrm2.cpp
+  blas1/rotg.cpp
+  blas1/rotm.cpp
+  blas1/rotmg.cpp
+  blas1/scal.cpp
+  #blas1/sdsdot.cpp -> not implemented in cuBLAS
+) 
+
+#if(${BLAS_ENABLE_EXTENSIONS})
+#  list(APPEND sources "extension/reduction.cpp")
+#endif()
+
```

Comment:
Thanks for your patch, @s-Nick!

```suggestion
```

Reply:
I let it commented because we may need it in the near future. Should I remove it?

Reply:
I don't think there are cuBlas extension we want to benchmark.